wrapanapi/systems/container/rhopenshift.py

```diff
@@ -297,6 +297,28 @@ def list_image_registry(self, namespace=None):
         # returns only the image registry name, without the port number in case of local registry
         return sorted(set([status.image.split('/')[0].split(':')[0] for status in statuses]))
 
+    def expose_db_ip(self, namespace):
+        """Creates special service in appliance project (namespace) which makes internal appliance
+           db be available outside.
+        Args:
```

Comment:
**REQUIRED**
Please put a blank line after the description to comply with the style guide.

Reply:
fixed